docs/guides/upgrading.rst

```diff
@@ -439,6 +439,14 @@ The metadata ``$entity->view`` no longer specifies the view used to render in ``
 
 Similarly the property ``$annotation->view`` no longer has an effect within ``elgg_view_annotation()``.
 
+API authentication for web services is required by default
+----------------------------------------------------------
+
+All exposed methods that do not explicitly set API auth requirement to false, will
+require API authentication.
+
```

Comment:
Could you add short instructions how to check the value and change it if necessary?